go.mod

```
@@ -103,7 +104,7 @@ require (
 	github.com/gogo/protobuf v1.3.1
 	github.com/golang/groupcache v0.0.0-20200121045136-8c9f03a8e57e
 	github.com/golang/mock v1.3.1
-	github.com/golang/protobuf v1.3.4
+	github.com/golang/protobuf v1.4.1
```

```suggestion
	github.com/golang/protobuf v1.4.3
```
Why not like this? And remove the `replace` clause